config/request-docs.php

```diff
@@ -58,13 +58,17 @@
         'description' => 'Laravel Request Docs',
         // default version that this library provides
         'version' => '3.0.0',
-        // changeable
+        //changeable
         'document_version' => '1.0.0',
         // license that you want to display
         'license' => 'Apache 2.0',
         'license_url' => 'https://www.apache.org/licenses/LICENSE-2.0.html',
         'server_url' => env('APP_URL', 'http://localhost'),
-
+        //openapi 3.0.x doesn't support request body for develop operation
```

Comment:
```suggestion
        // openapi 3.0.x doesn't support request body for develop operation
```

Reply:
```
... for develop operation
```
Do you mean `delete` ?

Reply:
sorry, that's a typo. yes for delete operation